### PR TITLE
Add confetti when winning

### PR DESCRIPTION
**Pull request by Ojself**
Add confetti
Fix skewered text caused by instructions

### CODE REVIEW

**Comment by Parnswir** [comment]
I don't see the confetti :-\

File: public/scripts/menu.js [hunk 1]
```diff
@@ -251,12 +253,13 @@ const continueGame = pid => {
 
 // generates a new game
 const createGame = (player1Name, player2Name) => {
+    const body = JSON.stringify( {"players": [player1Name || 'orange', player2Name || 'blue'] })
```
Comment (Parnswir) [inline public/scripts/menu.js:256]:
```suggestion
    const body = JSON.stringify({ players: [player1Name || 'orange', player2Name || 'blue'] })
```

File: public/shapes/hud.js [hunk 2]
```diff
@@ -19,9 +21,15 @@ class Hud {
 		];
 	}
 
+	setupConfetti(){
+		if (!this.winner)return
```
Comment (Parnswir) [inline public/shapes/hud.js:25]:
```suggestion
		if (!this.winner) return;
```

**Comment by Ojself** [comment]
It should be working now:)

**Comment by Parnswir** [comment]
Awesome!

**Pull request merged**